waltz-ng/client/technology/components/technology-summary-section.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Waltz - Enterprise Architecture
+ * Copyright (C) 2016  Khartec Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const bindings = {
+    stats: '<',
+    sourceDataRatings: '<'
+};
+
+
+const template = require('./technology-summary-section.html');
+
+
+function controller() {
+
+}
+
```

Comment:
controller is optional, you can have a 'dumb' component with just bindings and template.